cmd/db.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+)
+
+var dbSize, dbName, dbNetworkID, software, softwareVersion, firewallID string
+var replicas, numSnapshots int
+var publicIP bool
+
+// DBCmd is the root command for the db subcommand
+var DBCmd = &cobra.Command{
+	Use:     "db",
+	Aliases: []string{"database"},
+	Short:   "Manage Civo Database ",
+	Long:    `Create, update, delete, and list Civo Database.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := cmd.Help()
+		if err != nil {
+			return err
+		}
+		return errors.New("a valid subcommand is required")
+	},
+}
+
+func init() {
+	DBCmd.AddCommand(dbCreateCmd)
+	dbCreateCmd.Flags().StringVarP(&dbSize, "size", "s", "g3.medium", "the size of nodes to create.")
+	dbCreateCmd.Flags().StringVarP(&dbNetworkID, "network", "n", "default", "the name of the network to use in the creation")
+	// TODO: Should we call it database type? So the short hand would be "t"
+	dbCreateCmd.Flags().StringVarP(&software, "software", "w", "mysql", "the name of the software to use in the creation")
+	dbCreateCmd.Flags().StringVarP(&softwareVersion, "software-version", "v", "", "the version of the software to use in the creation")
```

Comment:
Mention in the description if not provided, latest version will be used. 
We also need to have a separate command to list all available versions for each DB